data_structures/queue/monotonic_queue.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from .double_ended_queue import Deque
```

Comment:
Could this use https://docs.python.org/3/library/collections.html#collections.deque or is there a special capability in the local version?

Reply:
Yes we can. And so that we can avoid using private attributes